lib/watcher.js

```diff
@@ -85,13 +86,13 @@ module.exports = class Watcher extends EventEmitter {
           // Only set _rebuildScheduled to false *after* the setTimeout so that
           // change events during the setTimeout don't trigger a second rebuild
           this._rebuildScheduled = false;
-          return this._build();
+          return this._build(path.join(root, filePath));
```

Comment:
I'm not sure what the intention originally was here, but due to the debounce this will only report one file, even if multiple change...

Reply:
Looks like this is how the ember version worked, only the first file is reported. This keeps compat, but TBH I think it should report all the changed files. 
Should we consider adding a `changedFiles` property to the result hash, and take `changedFiles[0]` to be `filePath` for BC?

Reply:
@oligriffiths yes, the current design is... silly. I'm totally on-board with future work exploring adding a more complete list of files that have changed.